Changed text in selection conform other page

sandervandevelde opened the pull request:
Just like https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-csharp-node-twin-getstarted , the same, more descriptive selection text is shown. (one page is a combination of two languages, that is reflected in the selection now)

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

sandervandevelde committed:
Changed text in selection conform other page

Just like https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-csharp-node-twin-getstarted , the same, more descriptive selection text is shown. (one page is a combination of two languages, that is reflected in the selection now)
